packages/protocol/contracts/ACE/ACE.sol

```diff
@@ -93,8 +124,8 @@ contract ACE {
             calldatacopy(add(memPtr, 0x104), _proofData, add(calldataload(_proofData), 0x20))
 
             // call our validator smart contract, and validate the call succeeded
-            
-            if iszero(staticcall(gas, validatorAddress, memPtr, add(calldataload(_proofData), 0x124), 0x00, 0x00)) {
+            switch iszero(staticcall(gas, validatorAddress, memPtr, add(calldataload(_proofData), 0x124), 0x00, 0x00))
```

Comment:
we can remove `iszero` and test for `case 0` in the switch statement (we know that the return value of `staticcall` is already a boolean)